<?xml version="1.0" ?>
<robot name="xpider_shield_top_base_link" xmlns:xacro="http://ros.org/wiki/xacro">
<xacro:include filename="$(find xpider_resources)/urdf/common_materials.xacro"/>
<xacro:include filename="$(find xpider_resources)/urdf/common_constants.xacro"/>

  <xacro:include filename="$(find xpider_descriptions)/models/base_link.xacro"/>

  <joint name="base_joint" type="fixed">
		<parent link="base_link" />
		<child link="shield_top" />
		<origin xyz="0 0 0" rpy="0 0 0" />
	</joint>

  <xacro:include filename="$(find xpider_descriptions)/models/shield_top.xacro"/>

  <joint name="base_joint2" type="fixed">
		<parent link="base_link" />
		<child link="shield_bottom" />
		<origin xyz="0 0 0" rpy="0 0 0" />
	</joint>

  <xacro:include filename="$(find xpider_descriptions)/models/shield_bottom.xacro"/>

  <joint name="base_joint3" type="fixed">
		<parent link="base_link" />
		<child link="step" />
		<origin xyz="0 0 0" rpy="0 0 0" />
	</joint>

  <xacro:include filename="$(find xpider_descriptions)/models/step.xacro"/>

  <joint name="base_joint4" type="fixed">
		<parent link="base_link" />
		<child link="step_base" />
		<origin xyz="0 0 0" rpy="0 0 0" />
	</joint>

  <xacro:include filename="$(find xpider_descriptions)/models/step_base.xacro"/>
</robot>
